src/parameter_declarations.ts

```diff
@@ -325,3 +327,14 @@ Parameter.declare({
 		'(OPTIONAL) Derives a drive key for the given drive ID without verifying its correctness against the drive on chain.',
 	type: 'boolean'
 });
+
+Parameter.declare({
+	name: LocalPathsParameter,
+	aliases: ['--local-paths'],
+	type: 'array',
+	description: `(BETA) a glob pattern to select files to be uploaded
```

Comment:
It's no longer a glob pattern, it's a list of files. We could add a note indicating that shell expansion is supported.